scripts/packages/chocolatey/build.ps1

```diff
@@ -18,8 +18,11 @@ rm -force ./bazel.nuspec
 rm -force ./tools/chocolateyinstall.ps1
 
 Add-Type -A System.IO.Compression.FileSystem
-$zipDir = "$pwd/../../../output"
+$outputDir = "$pwd/../../../output"
 $zipFile = "$pwd/$($tvFilename)"
+$zipDir = "$outputDir/package"
+new-item $zipDir -type directory -ErrorAction SilentlyContinue
+cp "$outputDir/bazel.exe" $zipDir
```

Comment:
Done in 642c535. If you'd prefer ZipArchive class usage instead of [compress-archive introducing powershell v5 dependency](https://technet.microsoft.com/en-us/library/dn841358(v=wps.640).aspx), please supply an example? I searched but didn't find one before finding that.